content/applications/sales/sales/products_prices/prices/pricing.rst

```diff
@@ -1,146 +1,438 @@
 
-Prices per country
-==================
+To configure the :guilabel:`Formula` computation option, start by selecting an option from the
+:guilabel:`Based on` field: :guilabel:`Sales Price`, :guilabel:`Cost`, or :guilabel:`Other
+Pricelist`. This determines what the advanced price rule formula will be based on.
 
-Pricelists can be set by countries group. Any new customer recorded in Odoo gets a default
-pricelist, i.e. the first one in the list matching the country. In case no country is set for the
-customer, Odoo takes the first pricelist without any country group.
+Next, in the :guilabel:`Discount` field, determine how much of a discount should be applied. It
+should be noted that a mark-up can be applied by setting a negative discount in this field.
 
-The default pricelist can be replaced when creating a sales order.
+.. example::
+   To formulate a 100% markup (or 2 times the cost of the product), with a $5 minimum margin, set
+   the :guilabel:`Based on` field to :guilabel:`Cost`, the :guilabel:`Discount` to `-100`, and the
+   :guilabel:`Margins` to `5`. This is often seen in retail situations.
 
-.. tip:: You can change the pricelists sequence by drag & drop in list view.
+   .. image:: pricing/formula-markup-cost-example.png
+      :align: center
+      :alt: How it looks to formulate a markup cost with 5 dollar minimum margin in Odoo Sales.
 
-Compute and show discount % to customers
-========================================
+Then, in the :guilabel:`Extra Fee` field, specify a fixed amount to add (or subtract) to the amount
+calculated with the discount. After that, enter a desired figure in the :guilabel:`Rounding Method`
+field. The rounding method sets the price so that it is a multiple of the value in the field.
 
-In case of discount, you can show the public price and the computed discount % on printed sales
-orders and in your eCommerce catalog. To do so:
+.. note::
+   Rounding is applied *after* the discount and *before* the surcharge.
 
-* Check *Allow discounts on sales order lines* in :menuselection:`Sales --> Configuration -->
-  Settings --> Quotations & Sales --> Discounts`.
-* Apply the option in the pricelist setup form.
+.. tip::
+   To have prices that end in 9.99, set the :guilabel:`Rounding Method` to `10` and the
+   :guilabel:`Extra Fee` to `-0.01`.
 
-.. image:: pricing/discount_options.png
-   :align: center
+Lastly, specify the minimum amount of margin over the base price in the :guilabel:`Margins` field.
 
-.. seealso::
+Once all formula-related configurations are complete, Odoo provides an example of the formula in a
+blue block to the right of the configurations.
+
+.. example::
+   To apply 20% discounts, with prices rounded up to 9.99, set the :guilabel:`Based on` field to
+   :guilabel:`Sales Price`, the :guilabel:`Discount` field to `20`, the :guilabel:`Extra Fee` field
+   to `-0.01`, and the :guilabel:`Rounding Method` field to `10`.
```

Comment:
I almost think you can remove this call out, since the image has an example. It could be a bit repetitive.

Reply:
I fully understand what you mean, but I've been instructed to leave such copy -- just in case the image doesn't load for any reason.